tests/vmi_configuration_test.go

```diff
@@ -2508,15 +2508,32 @@ var _ = Describe("Configurations", func() {
 		vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 		Expect(err).ToNot(HaveOccurred(), "Should successfully get VMI")
 
-		vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+		vmiVirtLauncherPod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
 
 		tests.WaitAgentConnected(virtClient, vmi)
 
-		// Ensure that VMI IP Stays equal to PodIP even after Guest-Agent kicks in
-		Consistently(func() bool {
+		// Ensure that VMI 'ipAddress' and 'ipAddresses' stays equal to
+		// VMI virt-launcher pod 'PodIP' and 'PodIPs' even after Guest-Agent kicks in
+		Consistently(func() error {
 			vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred(), "Should successfully get VMI")
-			return vmi.Status.Interfaces[0].IP == vmiPod.Status.PodIP
-		}, 30*time.Second, 1*time.Second).Should(BeTrue(), "VMI status IP should match VMI Pod IP")
+
+			if vmiVirtLauncherPod.Status.PodIP != vmi.Status.Interfaces[0].IP {
```

Comment:
looks like you're doing some very similar to `ValidateVMIandPodIPMatch` in [another PR](https://github.com/kubevirt/kubevirt/pull/4042) - can you maybe use that? no need for both implementations..